content/docs/concepts/traffic-management/overview/index.md

```diff
@@ -0,0 +1,155 @@
+Kubernetes adapter translates this data for the abstract model, so Pilot can
+generate and deliver the appropriate Envoy-specific configurations.
+
+The Pilot **service discovery and traffic rules** use the abstract model to let
+Envoy proxies know about one another in the mesh through the **Envoy API.**
+
+You can use the **Networking and Rules APIs** to exercise more granular control
+over the traffic in your service mesh.
+
+## Envoy proxies
+
+Traffic in Istio is categorized as data plane traffic and control plane
+traffic. Data plane traffic refers to the data that the business logic of the
+workloads manipulate. Control plane traffic refers to configuration and control
+data sent between Istio components to program the behavior of the mesh. When
+referring to traffic management in Istio, the traffic involved is exclusively
+data plane traffic.
+
+Envoy proxies are the only Istio components that interact with data plane
+traffic. Envoy proxies route the data plane traffic across the mesh and enforce
+the configurations and traffic rules without the services having to be aware of
+them. Envoy proxies mediate all inbound and outbound traffic for all services
+in the mesh. Envoy proxies are deployed as sidecars to services, logically
+augmenting the services with traffic management features, including the two
+discussed in this overview:
+
+-  [Service discovery](/docs/concepts/traffic-management/overview/#discovery)
+-  [Load balancing](/docs/concepts/traffic-management/overview/#load-balancing)
+
+The [traffic routing and configuration](/docs/concepts/traffic-management/routing/)
+and [network resilience and testing](/docs/concepts/traffic-management/network/)
+sections dig into more sophisticated features and tasks enabled by Envoy
+proxies, which include:
+
+-  Traffic management features: enforce fine-grained traffic control with rich
+   routing rules for HTTP, gRPC, WebSocket, and TCP traffic.
+
+-  Network resiliency features: setup retries, failovers, circuit breakers, and
+   fault injection.
+
+-  Security and authentication features: enforce security policies and enforce
+   access control and rate limiting defined through the configuration API.
+
+### Platform-agnostic service discovery {#discovery}
+
+Service discovery works in a similar way regardless of what platform you're
+using:
+
+1. The platform starts a new instance of a service and notifies its platform
+   adapter.
+
+1. The platform adapter registers the instance with the Pilot abstract model.
+
+1. **Pilot** distributes traffic rules and configurations to the Envoy proxies
+   to account for the change.
+
+The diagram shows how the platform adapters and Envoy proxies interact.
+
+{{< image width="40%"
+    link="./discovery.svg"
+    caption="Service discovery"
+    >}}
+
+Because service discovery feature is platform-independent:
+
+-  A service mesh can include services across multiple platforms.
+
+-  Envoy proxies perform service discovery.
```

Comment:
Do they?  How?

Reply:
They do. "How" is described in the Envoy docs that are outside the scope of this doc and linked above.

Reply:
I see it as "a platform adapter gets services from the service discovery method of something else.  Then, Pilot programs Envoys with those services".  Envoys don't _discover_ anything.

Reply:
Yes, Envoy has an API that it calls to discover service endpoints. Pilot implements this API (CDS). I would remove line 108.